python/pyrogue/_Block.py

```diff
@@ -15,6 +15,14 @@
 import pyrogue as pr
 
 
+def startTransaction(block, *, type, forceWr=False, checkEach=False, variable=None, index=-1, **kwargs):
+    block._startTransaction(type, forceWr, checkEach, variable, index)
+
+
+def checkTransaction(block, **kwargs):
```

Comment:
Should there be a print statement about how checkTransaction is depreciated and should use _checkTransaction instead?

Reply:
Same as above. The idea is the helper function will absorb and future proof the function args.